compiler-core/src/config.rs

```diff
@@ -836,6 +842,24 @@ impl Repository {
             Repository::Custom { .. } | Repository::None => None,
         }
     }
+
+    pub fn tag_for_version(&self, version: &Version) -> String {
+        let prefix = match self {
+            Repository::GitHub { tag_prefix, .. }
+            | Repository::GitLab { tag_prefix, .. }
+            | Repository::BitBucket { tag_prefix, .. }
+            | Repository::Codeberg { tag_prefix, .. }
+            | Repository::SourceHut { tag_prefix, .. }
+            | Repository::Gitea { tag_prefix, .. } => tag_prefix.as_ref(),
+
+            Repository::Custom { .. } | Repository::None => None,
+        };
+
+        match prefix {
+            Some(prefix) => format!("{prefix}{version}"),
```

Comment:
Should we add a `-` between them?

Reply:
It actually seems like a lot of people like to use `/` to put the tags in a directory. My thought was originally to give the freedom to the user to do what they like. However, if we'd use the package name automatically, then I think some delimiter would be good, and `/` would be a good candidate.